charts/keycloakx/examples/postgresql-ingress/keycloak-server-values.yaml

```
@@ -0,0 +1,78 @@
+# This is an example configuration, for production grade configuration see the Keycloak documentation.
+# See https://www.keycloak.org/server/configuration
+# See https://www.keycloak.org/server/all-config
+command:
+  - "/opt/keycloak/bin/kc.sh"
+  - "--verbose"
+  - "start"
+  - "--auto-build"
+  - "--http-enabled=true"
+  - "--http-port=8080"
+  - "--hostname-strict=false"
+  - "--hostname-strict-https=false"
+  - "--spi-events-listener-jboss-logging-success-level=info"
+  - "--spi-events-listener-jboss-logging-error-level=warn"
+  - "--proxy=edge"
+
+# Configure ingress
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    ingress.kubernetes.io/affinity: cookie
+  rules:
+    - host: id.acme.test
+      paths:
+        - path: /auth
+          pathType: Prefix
+  tls:
+    - hosts: 
+      - id.acme.test
+      paths:
+        - path: /auth
+          pathType: Prefix
+  console:
+    enabled: true
+    tls:
+      - hosts: 
+        - id.acme.test
+        paths:
```

the `paths` blocks need to be below a `rules` block instead of `tls`
```suggestion
     rules:
       - hosts:
          - id.acme.test
         paths:
```